tests/src/hipEnvVar.cpp

```diff
@@ -27,57 +27,61 @@ THE SOFTWARE.
 
```

Comment:
Is this test being run on CI?

Reply:
This test is used by hipEnvVarDriver.cpp which is run on CI.